Dockerfiles/dogstatsd/DOCKERHUB.md

```diff
@@ -0,0 +1,17 @@
+## Tags
+
+- `latest`
+- `7` , `7.X.X` : versions published since agent 7 has been released
```

Comment:
We should add a note in this page saying there's no breaking change, even though we bump the major version. This is just to be in sync with the agent version.